java8/src/main/java/org/jetbrains/annotations/BlockingContext.java

```diff
@@ -0,0 +1,12 @@
+package org.jetbrains.annotations;
```

Comment:
Please add a standard copyright header used in other files.

Reply:
sure! I see there is no copyright in the Blocking/NonBlocking annotations as well - should I add it there too?

Reply:
Yes, please.